packages/app-page-builder/src/admin/views/Pages/Pages.tsx

```diff
@@ -39,8 +37,16 @@ const Pages: React.FC = () => {
     const onCreatePage = useCallback(async (template?: PbPageTemplate) => {
         setIsLoading(true);
         try {
-            const res = await create({
-                variables: { category: "static" }, // hardcoded for now
+            const MUTATION = template ? CREATE_PAGE_FROM_TEMPLATE : CREATE_PAGE;
+            const variables = {
+                // category is temporarily hardcoded
+                category: "static",
```

Comment:
Double checking - should this stay hardcoded? Probably we need to create the UI before undoing this?